src/main/python/resources/checkers/input_version_checker.py

```diff
@@ -135,7 +135,7 @@ def _apply_export_version(self, argument, type_of_check):
         Function to set the global VEP version or GRCh build.
         :param argument: int or float
         """
-        if argument is not False:
+        if argument:
```

Comment:
This one is a bit different, as both arguments that come though here are either `None` or `int` / `float`. In this case, checking if a value is at least present (and maybe check if value is greater than 0). A value can't be "empty" (i.e. `''`), since that would be considered a string, which can't be set in `CapiceManager()` since it expects an `int` or `float` (or `None`)